web_external/js/utilities.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Miscellaneous utility functions.
+ */
+
+/**
+ * Show an alert dialog with a single button.
+ * @param [text] The text to display.
+ * @param [buttonText] The text for the button.
+ * @param [buttonClass] Class string to apply to the button.
+ * @param [escapedHtml] If you want to render the text as HTML rather than
+ *        plain text, set this to true to acknowledge that you have escaped any
+ *        user-created data within the text to prevent XSS exploits.
+ * @param callback Callback function called when the user clicks the button.
+ */
+isic.showAlertDialog = function (params) {
```

Comment:
Is there a reason this isn't a Backbone view, like [ImageFullscreenWidget](https://github.com/ImageMarkup/isic-archive/blob/master/web_external/js/views/widgets/ImageFullscreenWidget.js)?